Facebook-DP-Downloader/Readme.md

```diff
@@ -0,0 +1,22 @@
+# <center> Facebook Profile Picture Downloader </center>
+<center> Download the profile picture of any public profile on facebook by just it's fbid. </center>
+<br>
+
+## Modules used:
+---------------
+* os
+* request
```

Comment:
remove this section and add the requirements.txt file.  Ref this file for example: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt